src/components/images/imageLoader.js

```diff
@@ -87,9 +87,6 @@ import './style.css';
             requestAnimationFrame(() => {
                 if (elem.tagName !== 'IMG') {
                     elem.style.backgroundImage = "url('" + url + "')";
-                    if (elem.classList.contains('blurhashed')) {
-                        elem.style.backgroundColor = '#fff';
```

Comment:
I think this was done to fix transparency issues

Reply:
@nielsvanvelzen Yeah, I noticed now after tracking where the change came from. Pushed a commit that changes the canvas opacity, that should work. Can't test it as I don't have any image with transparency

Reply:
I can test. The change was made specifically for channel images for live tv.

Reply:
No idea when it changed, but it seems like blurhashes are no longer being generated for the channel logos. :tada: